### PR TITLE
Assets supports report

### PR DESCRIPTION
**Pull request by mkurek**

### CODE REVIEW

**Comment by ar4s** [comment]
:ok:

**Review by quamilek — APPROVED** [review]
test

**Review by quamilek — CHANGES_REQUESTED** [review]
tests

**Review by quamilek — COMMENTED** [review]
test

**Review by quamilek — APPROVED** [review]

File: src/ralph/reports/views.py [hunk 1]
```diff
@@ -366,6 +370,92 @@ def _get_tags(self, obj):
...
+            'baseobject__asset__property_of',
+            'baseobject__service_env__service',
+        ).prefetch_related(
+            # AttachmentItem is attached to BaseObject (by content type),
+            # so we need to prefetch attachments through base object of support
+            'support__baseobject_ptr__attachments__attachment'
+        )
+        headers = []
+        select_related = []
+        if model._meta.object_name == 'DataCenterAsset':
```
Comment (ghost) [inline src/ralph/reports/views.py:414]: Perhaps, this could be model method removing this if.

**Comment by ghost** [comment]
:ok:

**Pull request merged**